.github/workflows/cicd.yml

```diff
@@ -37,3 +37,36 @@ jobs:
             build/tmp/deploy/sdk/HULKs-OS-x86_64-toolchain-${{ github.ref_name }}.sh
             build/tmp/deploy/sdk/HULKs-OS-aarch64-toolchain-${{ github.ref_name }}.sh
           fail_on_unmatched_files: true
+  devimage:
```

Comment:
If you have an entirely new job that runs on an entirely different host. Can we merge the release steps? Such that there is one build job, that is run on our self-hosted runner and one release job, that downloads the artifacts of the build job and builds container images, and releases the binaries to github.